sdk/monitor/azure-monitor-opentelemetry-exporter/azure/monitor/opentelemetry/exporter/export/_base.py

```diff
@@ -322,9 +322,9 @@ def _transmit(self, envelopes: List[TelemetryItem]) -> ExportResult:
                             from azure.monitor.opentelemetry.exporter.statsbeat._statsbeat import shutdown_statsbeat_metrics
                             shutdown_statsbeat_metrics()
                             # pylint: disable=lost-exception
-                            return ExportResult.FAILED_NOT_RETRYABLE
+                            return ExportResult.FAILED_NOT_RETRYABLE  # pylint: disable=W0134
```

Comment:
Seems like the current version of pylint doesn't recognize W0134 and is failing on that. In order to get both pylint and next-pylint to pass, can just change this to `disable=W0012,W0134`